dali/kernels/test/warp_test/warp_cpu_test.cc

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
```

Comment:
I mentioned it before, but here it goes again: this `warp_test` directory is not consistent with what we do in the rest of DALI (keeping `*_test.*` files in the same directory as the implementation files).